# The phase durations

float64[]		poly_durations		# polynomials durations in the spline
float64[]		phase_durations		# phase durations of the motion
